test/hotspot/jtreg/runtime/appcds/MoveJDKTest.java

```diff
@@ -142,6 +156,24 @@ static void clone(File src, File dst) throws Exception {
         }
     }
 
+    static void renameModulesFile(String javaHome) throws Exception {
+        String modulesDir = javaHome + File.separator + "lib";
+        File origModules = new File(modulesDir, "modules");
+        if (!origModules.exists()) {
+            throw new RuntimeException("modules file not found");
+        }
+
+        File renamedModules = new File(modulesDir, "orig_modules");
+        if (renamedModules.exists()) {
+            throw new RuntimeException("found orig_modules unexpectedly");
+        }
+
+        boolean success = origModules.renameTo(renamedModules);
+        if (!success) {
+            throw new RuntimeException("rename modules file failed");
+        }
+    }
+
     static ProcessBuilder makeBuilder(String... args) throws Exception {
```

Comment:
Includes the changes from [JDK-8205946](https://bugs.openjdk.org/browse/JDK-8205946) ( commit [a10d6e7](https://github.com/openjdk/jdk/commit/a10d6e71d8b78563648b7fec63c8edc79c27cc77) ) on this file